__tests__/unit/configuration/configuration.test.js

```diff
@@ -688,10 +689,53 @@ describe('with version 1', () => {
     let context = createMockGhConfig(configString, prConfigString, files)
     context.eventName = 'pull_request'
 
+    let config = null
+
     process.env.USE_CONFIG_FROM_PULL_REQUEST = 'false'
-    let config = await Configuration.fetchConfigFile(context)
-    let parsedConfig = yaml.safeLoad(configString)
-    expect(config).toEqual(parsedConfig)
+    config = await Configuration.fetchConfigFile(context)
+    expect(config).toEqual(yaml.safeLoad(configString))
+
+    process.env.USE_CONFIG_FROM_PULL_REQUEST = 'true'
+    config = await Configuration.fetchConfigFile(context)
+    expect(config).toEqual(yaml.safeLoad(prConfigString))
+  })
+
+  test('that env USE_ORG_AS_DEFAULT_CONFIG correctly use org-wide config', async () => {
+    let repoConfig = `
+      version: 2
+      mergeable:
+        - when: pull_request.*, pull_request_review.*
+          name: 'repository rules'
+          validate:
+            - do: title
+              must_include:
+                regex: 'something'`
+
+    let orgConfig = `
+      version: 2
+      mergeable:
+        - when: pull_request.*, pull_request_review.*
+          name: 'organization rules'
+          validate:
+            - do: title
+              must_include:
+                regex: 'nothing'`
+
+    let context = createMockGhConfig(repoConfig, orgConfig)
+    context.event = 'pull_request'
+
+    let config = null
+
+    process.env.USE_ORG_AS_DEFAULT_CONFIG = 'true'
```

Comment:
There is an `.env.template` that is sorely outdated. Would be great if you can start keeping that updated by adding this env variable in with an explanation. We can resolve the other missing env variables separately.